frontend/src/metabase/static-viz/components/XYChart/shapes/LineSeries.tsx

```diff
@@ -29,17 +40,63 @@ export const LineSeries = ({
           return null;
         }
 
+        const yAccessor = (datum: SeriesDatum) => yScale(getY(datum)) ?? 0;
         return (
-          <LinePath
-            key={s.name}
-            data={s.data}
-            x={xAccessor}
-            y={d => yScale(getY(d)) ?? 0}
-            stroke={s.color}
-            strokeWidth={2}
-          />
+          <>
+            <LinePath
+              key={s.name}
+              data={s.data}
+              x={xAccessor}
+              y={yAccessor}
+              stroke={s.color}
+              strokeWidth={2}
+            />
+            {showValues &&
+              s.data.map((datum, index) => {
+                // Use the similar logic as presented in https://github.com/metabase/metabase/blob/3f4ca9c70bd263a7579613971ea8d7c47b1f776e/frontend/src/metabase/visualizations/lib/chart_values.js#L130
+                const showLabelBelow =
```

Comment:
Flip the data point value position using the same logic as
https://github.com/metabase/metabase/blob/3f4ca9c70bd263a7579613971ea8d7c47b1f776e/frontend/src/metabase/visualizations/lib/chart_values.js#L130-L134

#### Result
![image](https://user-images.githubusercontent.com/1937582/184912556-8f0f9741-4760-461a-9136-f582822be145.png)